libs/product/README.md

```diff
@@ -9,3 +9,58 @@ provides clear interfaces, models, and factories for the frontend of an ecommerc
 ```
 npm install @daffodil/product
 ```
+
+## Using the InMemory Driver
```

Comment:
I don't think we should ever assume a user will only use JUST the product in-mem. While it is possible, its also too confusing to explain the difference between one and the other. We should tend towards "singular" configuration policies vs. multiple.

Reply:
changed to just have the more complex version.

Reply:
done